presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/main/grouphome/GroupHomeScreen.kt

```diff
@@ -191,6 +222,39 @@ fun GroupHomeScreen(
     }
 }
 
+@Composable
+private fun TagFilter(
+    modifier: Modifier,
+    filterTagList: ImmutableList<FilterTagUiModel>,
+    onTagClicked: (FilterTagUiModel) -> Unit,
+) {
+    LazyRow(
+        modifier = modifier,
+        horizontalArrangement = Arrangement.spacedBy(6.dp),
+    ) {
+        item { Spacer(modifier = Modifier.width(10.dp)) }
```

Comment:
```suggestion
    LazyRow(
        modifier = modifier,
        horizontalArrangement = Arrangement.spacedBy(6.dp),
        contentPadding = PaddingValues(horizontal = 10.dp),
    ) {
```

요기 앞뒤로 패딩 주려고 넣어둔 것 같은데 이렇게도 할 수 있을듯!

Reply:
헐 맞아..
contentPadding 이 있구나! 대박 나이스~~